src/AST/Variable.cs

```diff
@@ -10,6 +10,7 @@ public override T Visit<T>(IDeclVisitor<T> visitor)
 
         public Type Type { get { return QualifiedType.Type; } }
         public QualifiedType QualifiedType { get; set; }
+        internal ExpressionObsolete Initializer { get; set; }
```

Comment:
Why is this internal? Just make it public and save your self all of the trouble.

Reply:
Probably because the type is named Obsolete. I can just make it public but I still need to generate the bindings anyways.

Reply:
Never mind that, it's only because we've been building a new automated tree of expressions which should over time replace these.